### DISCUSSION TITLE
Create plugin for MatterMost

### DISCUSSION THREAD

**Discussion by bdimitrovski** [ctrf-io/ctrf#12] [Reporters]
[As per discussion](https://github.com/ctrf-io/slack-ctrf/issues/2) with @Ma11hewThomas, it would be nice to see this work with Cypress and/or any other other modern testing framework and [MatterMost](https://mattermost.com/). 

Slack and MatterMost are already similar in many aspects, so it would be fairly easy to create a separate plugin only for MM.

I'd be happy to contribute with my knowledge of Cypress/Node/MM etc, as I have so far written something like this in regards to test reporting for MM:

<img width="1012" alt="Screenshot 2024-07-17 at 18 57 25" src="https://github.com/user-attachments/assets/c72f9323-9a42-43a4-a8ae-65438a55e4d2">

but I'd prefer combining all this information along with the CTRF standard too.

**Comment by Ma11hewThomas** [comment]
Thanks for the suggestion, the mattermost plugin has now been published here - https://github.com/ctrf-io/mattermost-ctrf

Feel free to propose changes
